consensus/src/dag/types.rs

```diff
@@ -275,6 +275,35 @@ impl TDAGMessage for Node {
     }
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Debug, Eq, Hash, Clone)]
+pub struct NodeId {
+    round: Round,
+    author: Author,
```

Comment:
Add epoch for cleaning purposes. 
Nit: does it make sense to have NodeMetadata be:
{
NodeId
Payload
digrest
}
?